File: app/src/main/java/com/neome/java/api/meta/base/dto/FormValueRaw.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.google.gson.JsonElement;
import com.neome.java.api.meta.base.Types.EntUserId;
import com.neome.java.api.meta.base.Types.MetaIdComp;
import com.neome.java.api.meta.base.Types.RowId;

import org.jetbrains.annotations.Nullable;

import java.util.Date;
import java.util.Map;

@SuppressWarnings("unused")
public class FormValueRaw
{
  @Nullable
  public EntUserId createdBy;

  @Nullable
  public Date createdOn;

  public RowId rowId;

  @Nullable
  public String rowOrder;

  @Nullable
  public EntUserId updatedBy;

  @Nullable
  public Date updatedOn;

  public Map<MetaIdComp, JsonElement> valueMap;
}

File: app/src/main/java/com/neome/api/meta/base/dto/FormValueRaw.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import kotlinx.serialization.json.JsonElement
import com.neome.api.meta.base.Types.EntUserId
import com.neome.api.meta.base.Types.MetaIdComp
import com.neome.api.meta.base.Types.RowId

interface FormValueRaw
{
  val createdBy: EntUserId?
  val createdOn: String?
  val rowId: RowId
  val rowOrder: String?
  val updatedBy: EntUserId?
  val updatedOn: String?
  val valueMap: Map<MetaIdComp, JsonElement>
}